### ISSUE TITLE
About miou performance with ptv2

### ISSUE THREAD

**Issue by dzp17** [Pointcept/Pointcept#306]
Thank you to the authors for making the code open-source. I encountered some issues while running ptv2. According to the config in your code, my results on the NuScenes validation set are consistent with those in your paper. However, the results on the test set are 79.7, while the reported results are 82.6. I would like to know if there might be something I am doing incorrectly or if there are any additional small tricks during testing. I have already used TTA (Test Time Augmentation).

**Comment by Gofinge** [comment]
Hi, train three PTv2 separately and vote on the prediction on one submission, which will deliver a better test performance. We encourage not using this trick for validation but only for testing on benchmarks, as many existing works already use the tech to boost performance.
